packages/cells/src/widget.ts

```diff
@@ -1593,6 +1593,7 @@ export class CodeCell extends Cell<ICodeCellModel> {
     }
   }
 
+  private _lastOutputHeight = '';
```

Comment:
Can you move this to around line 1667 where all other private variables are defined?